README.md

```diff
@@ -318,6 +318,16 @@ A function that flushes all existing loggers that are instantiated by createLogg
 ---
 A function that flushes all existing loggers that are instantiated by createLogger, and then removes references to them. Should only be called when you are finished logging.
 
+## Ambient Meta Support
+
+### addProperty(key, value)
+---
+A function that adds `key`/`value` pair into the `line`.
```

Comment:
I'd remove the escaped `line` and just put "line," as that's how it appears throughout the README currently.